src/fixture.ts

```diff
@@ -0,0 +1,18 @@
+import type { ZodTypeAny, z } from 'zod';
+import type { Config } from './core/core';
+import { Core } from './core/core';
+import type { Definition } from './core/generator';
+import defaultGenerators from './generators/default';
+
+export function createFixture<TSchema extends ZodTypeAny>(
+	schema: TSchema,
+	config: Config & {
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		extend?: Definition<any>[];
```

Comment:
Was your intention to do:
```ts
extend?: Definition<any> | Definition<any>[]
```

Reply:
Oh I haven't thought of this.
It was mainly for the `register` method.

Do you think we also need to do this for `extend`?

Reply:
Your call. `extend` is just the input for `register`.  I just assumed since you wanted `register` to accept a single generator that it made sense for `extend` to do the same.